flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/operations/DescribeCatalogOperation.java

```diff
@@ -84,7 +84,8 @@ public TableResultInternal execute(Context ctx) {
                                         "type",
                                         properties.getOrDefault(
                                                 CommonCatalogOptions.CATALOG_TYPE.key(), "")),
-                                Arrays.asList("comment", catalogDescriptor.getComment())));
+                                Arrays.asList(
+                                        "comment", catalogDescriptor.getComment().orElse(""))));
```

Comment:
Found a typo here, it should be `catalogDescriptor.getComment().orElse(null)`.

Reply:
I have updated as you said